src/DynamoUtilities/Md2Html.cs

```diff
@@ -76,6 +79,81 @@ internal string ParseMd2Html(string mdString, string mdPath)
             return GetData(processCommunicationTimeoutms);
         }
 
+        private void EncodeMDContent(ref string mdContent)
```

Comment:
How about making these two functions here also static?

Reply:
This functions were also changed to be static.
commit: https://github.com/DynamoDS/Dynamo/pull/15377/commits/1d7cd8d0afae6f3efa75f9e50509c28ae88ad8e5